doc/topics/proxyminion/index.rst

```diff
@@ -36,6 +36,21 @@ or more minions.
 See :ref:`Proxyminion Beacon <proxy-minion-beacon>` to help
 with easy configuration and management of ``salt-proxy`` processes.
 
+New in Nitrogen
```

Comment:
I have added also the under the Nitrogen release notes: https://github.com/saltstack/salt/pull/40249/commits/14923b6fde9c4833ce2207ad8e0ef9650eebd1ca